src/rospkg/environment.py

```diff
@@ -109,7 +109,7 @@ def get_ros_package_path(env=None):
     """
     if env is None:
         env = os.environ
-    return env.get(ROS_PACKAGE_PATH, None)
+    return env.get(ROS_PACKAGE_PATH, '') + ':/usr/share'
```

Comment:
You could consider doing this like:

```python
return env.get(ROS_PACKAGE_PATH, ':/usr/share')
```

The upside is that you could use rospkg without considering `/usr/share` by manually setting the `ROS_PACKAGE_PATH`. The downside is that people who set the `ROS_PACKAGE_PATH` need to make sure to add `/usr/share` to it when they're using packages from there.

I think it makes more sense for the Debian packages to set the `ROS_PACKAGE_PATH` to by inserting it into some shell rc file or something. That way there is visibility into what is being searched. Also, it would prevent the inconsistency between `rospkg` (Python) and `rospack` (C++).